content/en/docs/community/release_checklist.md

```diff
@@ -126,31 +158,34 @@ git tag --sign --annotate "$RELEASE_NAME" --message "Helm release $RELEASE_NAME"
 git push upstream "$RELEASE_NAME"
 ```
 
-This new tag is going to be the base for the patch release.
+This new tag is going to be the base for the patch release. Creating a
+[helm/helm milestone](https://github.com/helm/helm/milestones) is optional for
+patch releases.
 
 Make sure to check [helm on CircleCI](https://circleci.com/gh/helm/helm) to see
-that the release passed CI before proceeding.
+that the release passed CI before proceeding. Patch releases can skip steps 2-6
+and proceed to step 7 for [PGP signature](#7-pgp-sign-the-downloads).
 
-## 2. Change the Version Number in Git
+## 2. Major/Minor releases: Change the Version Number in Git
 
-When doing a minor release, make sure to update pkg/version/version.go with the
-new release version.
+When doing a major or minor release, make sure to update
+`internal/version/version.go` with the new release version.
```

Comment:
As far as I can tell, major releases would also need this to be done. Unsure why this said only "minor".